### ISSUE TITLE
does it not run on normal Arduino Uno with 16MHz ?

### ISSUE THREAD

**Issue by mega-hz** [c0pperdragon/SIO2SD#3]
Hi,

is it right that this doesn't run on 16MHz?
If so, could you please correct the timers for running on staandart 16MHz?

**Comment by c0pperdragon** [comment]
I don't think you need to change anything with the timings and run with 16MHz just fine. The timings are normally adjusted by the Arduino libraries accordingly.
8 MHz was just the highes speed available on a bare chip without external oscillator.

**Comment by mega-hz** [comment]
ah, ok.
would it be possible to remove sd part and just listen to sio devices and commands?
Say, if D1 is reading, turn on a led, if writing, turn on another led....

**Comment by c0pperdragon** [comment]
Brobably, but for this kind of bus-sniffing, the current code is a real overkill.
Maybe you want to do you own research on how to program such a thing from scratch. Here is a good resource for the protocol details: 
https://www.atarimax.com/jindroush.atari.org/asio.html

**Comment by mega-hz** [comment]
yes i know this site. just thought why do the whole work again. Your Sio Part should be run only, i will try.

**Comment by mega-hz** [comment]
but stilll a nice project!
Even i prefere an display instead of 7Seg.

**Comment by mega-hz** [comment]
i've got it running!
I only need your "handle-sio" routine.
If you don't mind i would like to use this in another Device.
SIO Device,Commands are shown on a LCD, works great!

**Comment by c0pperdragon** [comment]
very nice. 
and sure, use the code wherever you like
